src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+const CATEGORIES_STATUS = 'CATEGORIES_STATUS';
```

Comment:
- [ ] [REQUIRED] Kindly refactor the value of your Action Type to use the [ducks pattern](https://github.com/erikras/ducks-modular-redux):
`const CATEGORIES_STATUS = bookstore/categories/CATEGORIES_STATUS`